mhkit/wave/io/hindcast/hindcast.py

```diff
@@ -480,7 +492,11 @@ def request_wpto_directional_spectrum(
             },
         )
 
-    handle_caching(hash_params, cache_dir, data, meta)
+    handle_caching(
+        hash_params,
+        cache_dir,
+        cache_content={"data": data, "metadata": meta, "write_json": None},
+    )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args